rules-tests/Php81/Rector/Class_/SpatieEnumClassToEnumRector/Fixture/some_class.php.inc

```diff
@@ -26,6 +27,7 @@ enum StatusEnum : string
     case DRAFT = 'draft';
     case PUBLISHED = 'published';
     case ARCHIVED = 'archived';
+    case NEEDSREVIEW = 'needsReview';
```

Comment:
Could you add this as a new test fixture? So we have it separated and linked to specific issue.